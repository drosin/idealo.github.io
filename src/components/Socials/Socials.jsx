import React from 'react';
import './Socials.scss';

var FontAwesome = require('react-fontawesome');

export default (props) => {
    return (
        <p className={'socials__container ' + props.className}>
            <a href='https://www.xing.com/companies/axelspringerse' className='socials__item'><FontAwesome name='xing' /></a>
            <a href='https://de.linkedin.com/company/axel-springer-se' className='socials__item'><FontAwesome name='linkedin' /></a>
            <a href='https://www.instagram.com/axel_springer_karriere/' className='socials__item'><FontAwesome name='instagram' /></a>
            <a href='https://www.facebook.com/AxelSpringerKarriere/' className='socials__item'><FontAwesome name='facebook' /></a>
        </p>
    );
}