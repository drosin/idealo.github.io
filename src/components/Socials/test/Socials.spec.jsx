import React from 'react';
import renderer from 'react-test-renderer';
import Socials from '../Socials';

describe('Socials component', () => {
    it('should render as expected', () => {
        const renderedComponent = renderer.create(
            <Socials />
        );
        expect(renderedComponent.toJSON()).toMatchSnapshot();
    })
});