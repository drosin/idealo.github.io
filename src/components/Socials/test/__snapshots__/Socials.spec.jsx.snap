// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Socials component should render as expected 1`] = `
<p
  className="socials__container undefined"
>
  <a
    className="socials__item"
    href="https://www.xing.com/companies/axelspringerse"
  >
    <span
      aria-hidden={true}
      className="fa fa-xing"
    />
  </a>
  <a
    className="socials__item"
    href="https://de.linkedin.com/company/axel-springer-se"
  >
    <span
      aria-hidden={true}
      className="fa fa-linkedin"
    />
  </a>
  <a
    className="socials__item"
    href="https://www.instagram.com/axel_springer_karriere/"
  >
    <span
      aria-hidden={true}
      className="fa fa-instagram"
    />
  </a>
  <a
    className="socials__item"
    href="https://www.facebook.com/AxelSpringerKarriere/"
  >
    <span
      aria-hidden={true}
      className="fa fa-facebook"
    />
  </a>
</p>
`;
